docs/tutorials/quick-start.txt

```diff
@@ -307,7 +307,7 @@ experiment configuration file. The ``adaptive.yaml`` file tells the
 explore for each hyperparameter. For our model, we will use a fixed
 batch size and search on the dropout size, filters, and learning rate.
 In the ``searcher`` field, we update the ``name`` to use
-``adaptive_simple`` and add ``max_trials`` to use ``16``. This tells
+``adaptive_asha`` and add ``max_trials`` to use ``16``. This tells
```

Comment:
somehow I didn't realize until just now how easy to configure adaptive_asha is.  That's so cool.